README.md

```diff
@@ -94,6 +95,15 @@ which implores us to crash when we encounter bugs instead of limping along.
    signal. **Regardless of the action, the JVM is then forcibly killed via a
    `SIGKILL`.**
 
+In addition, `jvmquake` supports keyword arguments passed as comma separated
+`key=value` pairs in a fourth argument, so `int,int,int,<kwarg string>`:
+ * `warn` (type: int, default: maxint): an amount of GC "deficit" which will
+   cause `jvmquake` to touch a file (see `touch`) before it kills the JVM. The
+   default setting is effectively "off".
```

Comment:
"effectively" is a technicality that may confuse a user. Can we just say "the default is not to warn"?

Reply:
Done.